napari/_qt/dialogs/qt_reader_dialog.py

```diff
@@ -67,9 +67,25 @@ def setup_ui(self, error_message, readers):
 
         # checkbox to remember the choice (doesn't pop up for folders with no extension)
         if self._extension:
-            self.persist_checkbox = QCheckBox(
-                f'Remember this choice for files with a {self._extension} extension'
-            )
+            if (
+                '*' + self._extension
```

Comment:
this long statement in the `if` is leading to some not great formatting here. Could we use the dictionary `get` method to just save the preference into a variable directly e.g.

```python
existing_pref = get_settings().plugins.extension2reader.get('*' + self._extension)
if existing_pref:
    warn_message = ...
```